Cargo.toml

```diff
@@ -56,9 +56,9 @@ tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
 url = { version = "2.5.2", features = ["serde"] }
 
 reth-primitives = { git = "https://github.com/sp1-patches/reth", default-features = false, branch = "sp1-reth" }
-sp1-core-machine = "3.1.0"
-sp1-sdk = "3.0.0"
```

Comment:
ah we are tied to `v3.0.0` in the infrastructure for now (infra as in contracts in testnet and proving cluster)

Reply:
Ok, will revert the sp1 updates.